<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CharacterList</class>
 <widget class="QDialog" name="CharacterList">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>477</width>
    <height>507</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Character List</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QTreeWidget" name="treeWidget">
     <attribute name="headerVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string notr="true">1</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="btAddToTrack">
       <property name="text">
        <string>Add to tracking</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btAddCategory">
       <property name="text">
        <string>Add category</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btDelCategory">
       <property name="text">
        <string>Delete category</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btAddCharacter">
       <property name="text">
        <string>Add character</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btEditCharacter">
       <property name="text">
        <string>Edit character</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btDelCharacter">
       <property name="text">
        <string>Delete Character</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btImportCategory">
       <property name="text">
        <string>Import Category</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btExportCategory">
       <property name="text">
        <string>Export Category</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btClose">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btClose</sender>
   <signal>clicked()</signal>
   <receiver>CharacterList</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>527</x>
     <y>283</y>
    </hint>
    <hint type="destinationlabel">
     <x>297</x>
     <y>152</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
